src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -292,7 +292,7 @@
         </div>
 
         <!-- Count -->
-        <div *ngIf="!serviceForm.controls.unmanaged.value"
+        <div *ngIf="!serviceForm.controls.unmanaged.value && serviceForm.controls.service_type.value !== 'nvmeof'"
```

Comment:
why are we hiding the count in the case of nvmeof?

Reply:
It was suggested in discussions with nvme team and Paul.
This is of no use actually. Quoting what @pcuzner mentioned earlier on mockups:
>  count makes no sense here - at least for 8.0. one host will only support one gateway daemon. count is used to scale up stateless daemons like rgw

I missed to remove this in previous implementation